apps/docs/content/components/buttons/Button.mdx

```diff
@@ -124,4 +124,6 @@ A button can be rendered as a react router link when using the href property and
 
 ## Migration Notes
 
+Coming from Orbiter you should be aware of the following changes:
```

Comment:
It would be preferable to include this comment in the migration-notes.md file, as developers consulting it via GitHub or in the package won't have the necessary context.